knowledge_base/bundle_variables/README.md

```diff
@@ -0,0 +1,65 @@
+# bundle_variables
+
+Example to show how bundle variables work.
+
+We will focus on the behavior of the variables section alone. To do so, we exclusively use
+the `validate` command, and use its JSON output mode to focus on the variables section.
+
+## Usage
+
+Configure the workspace to use:
+```sh
+export DATABRICKS_CONFIG_PROFILE="<workspace profile>"
+```
+
+Try to run validate without arguments:
```

Comment:
```suggestion
Next, run `bundle validate` using the JSON output mode and `jq` to retrieve just the variables node.
```

Reply:
I wasn't sure how this command was "without arguments" but maybe I'm misunderstanding the intent there. Also, I'm not sure if it's necessary to state that last bit as most devs know all about jq, but it feels most complete.